src/services/preparePasteEdits.ts

```diff
@@ -25,6 +32,12 @@ export function preparePasteEdits(
             ancestorNode => rangeContainsRange(ancestorNode, range),
         );
         if (!enclosingNode) return;
+        if (

```

Comment:
Do all these conditions work if only a an identifier is selected? What about part of the module specifier? The test only shows the case where an entire import declaration is selected.

Reply:
Yes, if only an identifier is selected, it works too because it doesn't enter this if block, instead tries to enter the `forEachChild` but since it doesn't have any children it just returns and moves to the next iteration. Same for moduleSpecifier. If this is selected `{ foo }`, then it will enter the if block.

Reply:
Can you add tests to show that it works?